Makefile

```diff
@@ -281,7 +291,7 @@ preview-man: man
 
 .PHONY: test
 test: DEBUGFLAGS += -DDEBUGLEVEL=1
-test: all namespaceTest check test-xxhsum-c c90test test-tools
+test: all namespaceTest check test-xxhsum-c c90test test-tools test-unicode
```

Comment:
How will the test itself behave on systems which do not support unicode (if they exist) ?

If there are risks, and this test should rather be reserved to a few "good enough" environments, such as Windows, and I assume Linux and OS-X, then it should rather belong to `test-all` category.
`make test` is reserved for tests which are supposed to be "reasonably" universal

Reply:
Fair point. I'll move it.